ashrithashetty

```diff
@@ -0,0 +1,16 @@
+//Program to find the sum of numbers in a given range in C++
```

Comment:
Very Good. Appreciate your comment on what the program is doing.
Also leave a comment block about the file created date and author something like this :

/*
Created on October 22, 2018
@author: anoopbhatn
*/